.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Create a report to help us improve GNU Radio
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# 1. Have you looked in our documentation and asked the community?
+
+<!--- [Before you open an issue we suggest checking the following: -->
+<!--- - [The wiki](https://wiki.gnuradio.org/index.php/Main_Page) -->
+<!--- - [The GNU Radio Blocks documentation](https://wiki.gnuradio.org/index.php/Category:Block_Docs) -->
+<!--- - [The User manual](https://wiki.gnuradio.org/index.php/Usage_Manual) -->
+<!--- - [GNU Radio community](https://wiki.gnuradio.org/index.php/Chat) -->
+<!--- - or on [the mailing list](https://wiki.gnuradio.org/index.php/MailingLists)] -->
+
+# 2. GNU Radio version
```

Comment:
You might want to add a hint: `gnuradio-config-info --version`.
Or the GRC way with Help -> About (not entirely sure if this is correct though).

Reply:
>  Or the GRC way with Help -> About

That's correct (at least on OS X and Ubuntu)